src/cf-core/src/cf-base.less

```diff
@@ -556,14 +556,16 @@ select[multiple] {
     border: 1px solid @input-border;
     border-radius: 0;
     vertical-align: top;
-    -webkit-appearance: none;
+    appearance: none;
+    // TODO: update to use CSS 3 Basic UI specs as user-modify is deprecated.
     -webkit-user-modify: read-write-plaintext-only;
+    -moz-user-modify: read-write-plaintext-only;
```

Comment:
Oops, you're right, I misread that footnote in the docs. Sure, I can remove them both now.